src/NUnitFramework/testdata/TestFixtureSourceData.cs

```diff
@@ -72,10 +72,7 @@ public StaticProperty_SameClass(string arg, string expected) : base(arg, expecte
         {
         }
 
-        public static object[] StaticProperty
-        {
-            get { return new object[] { new object[] { "StaticPropertyInClass" } }; }
-        }
+        public static object[] StaticProperty => new object[] { new object[] { "StaticPropertyInClass" } };
```

Comment:
Please revert this change as it is not related to the changes for the `RepeatAttribute`

Reply:
reverted changes